openshift/template.yaml

```diff
@@ -586,4 +586,4 @@ objects:
                   address:
                     socket_address:
                       address: 127.0.0.1
-                      port_value: 8090
+                      port_value: 8090
```

Comment:
why do we need this change?

Reply:
look's the same, this may  accrued during the rebase - I'll check it out